.codespellignore

```diff
@@ -8,3 +8,10 @@ fpr
 falsy
 assertIn
 nD
+te
+Fo
+fo
+COO
+Onl
+Te
+ND
```

Comment:
Sorry, this is a bit nitpicky, but could you explain why all these “words” need to be added?

Reply:
Okay so the codespell is flagging these words as errors they are present in the notebook, i think they are the part of image data that is used. 
![Screenshot 2024-12-14 at 19-29-46 ](https://github.com/user-attachments/assets/b8e1628b-df6f-4a9d-afda-abab679a2e7b)
here "COO" is considered as a word so it it flagged by codespell and thus the branch fails the pre-commit test.
If there is another workaround this I will gladly update the branch

Reply:
Okay, that clears up a lot.

I would propose just cleaning the cell outputs.